### PR TITLE
[OSDOCS-11996]: Restructuring recommended etcd practices docs

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.17

Issue: https://issues.redhat.com/browse/OSDOCS-11996

Link to docs preview: https://81737--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.html

QE review:
- [x] QE has approved this change.

**NOTE TO PEER REVIEWERS:** This PR restructures the "Recommended etcd practices" topic to better align with the modular documentation guidelines. The biggest change was moving the procedure about verifying hardware to its own procedure module. Very little text changed.

### CODE REVIEW

**Review by kcarmichael08 — COMMENTED** [review]
Couple comments, looks good - much easier to find the info with the new structure!

File: modules/etcd-verify-hardware.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+** If you use Docker, run this command:
+[source,terminal]
++
+----
+$ sudo docker run --volume /var/lib/etcd:/var/lib/etcd:Z quay.io/cloud-bulldozer/etcd-perf
+----
+--
+
+The output reports whether the disk is fast enough to host etcd by comparing the 99th percentile of the fsync metric captured from the run to see if it is less than 10 ms. A few of the most important etcd metrics that might affected by I/O performance are as follow:
```
[the remark this answers is not in the document]
Comment (lahinson) [reply modules/etcd-verify-hardware.adoc:35]: Agree. Good catch!

File: modules/etcd-verify-hardware.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+* `etcd_disk_wal_fsync_duration_seconds_bucket` metric reports the etcd's WAL fsync duration
+* `etcd_disk_backend_commit_duration_seconds_bucket`  metric reports the etcd backend commit latency duration
+* `etcd_server_leader_changes_seen_total` metric reports the leader changes
+
+Because etcd replicates the requests among all the members, its performance strongly depends on network input/output (I/O) latency. High network latencies result in etcd heartbeats taking longer than the election timeout, which results in leader elections that are disruptive to the cluster. A key metric to monitor on a deployed {product-title} cluster is the 99th percentile of etcd network peer latency on each etcd cluster member. Use Prometheus to track the metric.
+
+The `histogram_quantile(0.99, rate(etcd_network_peer_round_trip_time_seconds_bucket[2m]))` metric reports the round trip time for etcd to finish replicating the client requests between the members. Ensure that it is less than 50 ms.
```
[the remark this answers is not in the document]
Comment (lahinson) [reply modules/etcd-verify-hardware.adoc:43]: I'll take a look and see if I can do anything to make it better.

**Pull request merged**